### ISSUE TITLE
Csc.dll should be moved from x64 -> AnyCPU

### ISSUE THREAD

**Issue by Sridhar-MS** [dotnet/roslyn#8951]
Here is the corflags output of csc.dll

```
C:\Program Files (x86)\Microsoft Visual Studio 14.0>corflags D:\github\cli\.dotnet_stage0\Windows\cli\bin\csc.dll
Microsoft (R) .NET Framework CorFlags Conversion Tool.  Version  4.6.81.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Version   : v4.0.30319
CLR Header: 2.5
PE        : PE32+
CorFlags  : 0x9
ILONLY    : 1
32BITREQ  : 0
32BITPREF : 0
Signed    : 1
```

**Comment by davkean** [comment]
What's the actual bug or ask here? It's not clear. Are you saying it should be marked 32-bit? It should be marked AnyCPU? We should ship two versions ?

**Comment by Sridhar-MS** [comment]
It should be marked as `AnyCPU` and ship one version assuming everything is purely managed. I am working on creating 32 bit [dotnet-cli](https://github.com/dotnet/cli) tools and running csccore using x86 coreclr fails today.

**Comment by agocke** [comment]
It looks like this was resolved with #9033
